### ISSUE TITLE
Extreme oligozoospermia

### ISSUE THREAD

**Issue by MagoWyr** [obophenotype/human-phenotype-ontology#9801]
**Preferred term label:**
Extreme oligozoospermia

**Synonyms**

**Definition (free text, please give PubMed ID)**
>0 - <2 million sperm (>0 - <1 million/mL concentration)

**Parent term (use hpo.jax.org/app)**
HP:0000798

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA: 399805

**Your nano-attribution (ORCID)**
ORPHA: 399805

**Comment by pnrobinson** [comment]
@MagoWyr 
I do not understand the proposed definition -- is the first number (0 - <2 million sperm) for the entire ejaculate? Logically it would need to be >0 to match with the concentration. What about the following definition:

A type of oligospermia characterized by a total number of more than zero but less than 2 million sperm in the ejaculate or a concentration of more than zero but less than one million sperm per milliliter.

**Comment by MagoWyr** [comment]
Yes, the first number is for the entire ejaculate. So it represents the absolute sperm count.
>0 - <2 million sperm (>0 - <1 million/mL) is the correct labelling. Sorry if this was not correct in my request.
Your definition "A type of oligospermia characterized by a total number of more than zero but less than 2 million sperm in the ejaculate or a concentration of more than zero but less than one million sperm per milliliter." is in general correct, but please note that the term is "oligoZOOspermia" and not "oligospermia".
________________________________

**Comment by pnrobinson** [comment]
@MagoWyr 
added as
HP:0034815
